packages/example-next-13-pages-and-app/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "example-next-13-pages-and-app",
+  "version": "2.9.1",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "lint": "eslint src && tsc",
+    "test": "playwright test",
+    "build": "next build",
+    "start": "next start"
```

Comment:
For this new package to be considered in the monorepo, it needs to be added to `package.json` in the root as well as `lerna.json`. After that you can add the build job to [the CI workflow](https://github.com/amannn/next-intl/blob/main/.github/workflows/main.yml).